.github/workflows/pullpreview.yml

```diff
@@ -45,7 +45,7 @@ jobs:
       with:
         admins: crohr,HDinger,machisuji,oliverguenther,ulferts,wielinde,cbliard
         compose_files: docker-compose.pullpreview.yml
-        instance_type: large
+        instance_type: xlarge
```

Comment:
@Kharonus Do you intend to have that change for all pull previews? Or just for this PR? Maybe you need to remove it here again?